grafana-plugin/src/containers/ApiTokenSettings/ApiTokenSettings.tsx

```diff
@@ -67,6 +67,16 @@ class ApiTokens extends React.Component<ApiTokensProps, any> {
       },
     ];
 
+    const requiredPermissionToView = UserActions.APIKeysWrite;
+    const authorizedToViewAPIKeys = isUserActionAllowed(requiredPermissionToView);
+
+    let emptyText = 'Loading...';
+    if (!authorizedToViewAPIKeys) {
+      emptyText = `${generateMissingPermissionMessage(requiredPermissionToView)} to be able to view API tokens.`;
+    } else if (apiTokens) {
+      emptyText = 'No tokens found';
```

Comment:
Not sure if there's any convention on oncall, but all phrases should end with `.`